app/views/common/SPKeyboardAvoidingView.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { KeyboardAvoidingView, Platform, StyleSheet } from 'react-native';
+
+export const SPKeyboardAvoidingView = ({ behavior, children }) => {
+  const isiOS = Platform.OS === 'ios';
```

Comment:
I think it would be useful to just define this function once and for all in `helpers/General.js`. We have a bunch of different functions in various places that all do the same check. 

Could you just define two `consts` in that file as part of this PR?

```js
const isAndroid = Platform.OS === 'android';
const isIOS = Platform.OS === 'ios'
```

Reply:
did some searching and looks like we already have those functions in Utils.js